Geometry/MTDCommonData/data/etl/v4/etl.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0"?>
-<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../../DetectorDescription/Schema/DDLSchema.xsd">
+<DDDefinition>
 
   <ConstantsSection label="etl.xml" eval="true/false">
-    <Constant name="ETLrmin" value="[caloBase:Rmin100]"/>   <!-- 26.3 cm -->
+    <Constant name="ETLrmin" value="[caloBase:Rmin100]"/>   <!-- 28.5 cm -->
     <Constant name="ETLrmax" value="[caloBase:Rmax100]"/>   <!-- 125.525 cm -->
-    <Constant name="ETLthickness" value="7.7"/>
```

Comment:
@bsunanda thanks for updating that, there was a cleaning of v5 (I13) but not of this earlier version